frontend/ui/bidi.lua

```diff
@@ -0,0 +1,157 @@
+--[[--
+Bidirectional text and UI mirroring helpers.
+Some overview at:
+https://material.io/design/usability/bidirectionality.html
+]]
+
+local Language = require("ui/language")
+local _ = require("gettext")
+
+local Bidi = {
+    _mirrored_ui_layout = false,
+    _rtl_ui_text = false,
+}
+
+-- Setup UI mirroring and RTL text for UI language
+function Bidi.setup(lang)
+    local is_rtl = Language:isLanguageRTL(lang)
```

Comment:
I assume you didn't use `isRtlLanguage` because it looks a bit odd (as does `isRTLLanguage) but I'd still prefer it for easier comprehension.

Reply:
Yes, flat camelCAse is a a bit strange.
But also, it puts the accent on what's checked for at the end: IsThisLanguageRTL., while your proposal sounds like IsItSomeRtlLanguage (among others) - less obvious.

Reply:
That depends on if you think of it as more of a query or more of a boolean, I suppose. As a natural language question I'd agree with you, but I see it in line with isKobo and isTouchDevice.

```lua
if isKobo() then
if isRtlLanguage() then
if isLanguageRTL() then
```

Reply:
But these Device: methods refer to the Device object they are methods of, so Device:isKobo() makes sense - a boolean/property.
Language here is a module, not an object, so Language:isLanguageRTL(lang) does not refer to a property/boolean of the Language - but it's a function that takes an argument and should tell if that argument is RTL or not - a query.
Anyway, it's only used at 2 places, so I don't mind a name that feel strange to me - if you still feel isRtlLanguage is preferable (`isRTL(lang)` could work, but it sound even more like a method than the 2 other longer solutions).